python/GafferSceneUI/RenderPassEditor.py

```diff
@@ -673,6 +717,13 @@ def __buttonPress( self, pathListing, event ) :
 					"shortCut" : "D",
 				}
 			)
+			menuDefinition.append(
+				"Select Affected Objects",
+				{
+					"command" : functools.partial( self.__selectAffected, pathListing ),
+					"active" : len( self.__selectedSetExpressions( pathListing ) ) > 0,
```

Comment:
I know it's often good to keep menu layouts consistent for muscle memory, but I think maybe it's not so helpful in this case - I think it would be clearer to omit the item rather than disable it. It seems strange to show it greyed out when you've got AA samples selected for example - it's not like there is some circumstance in which it will later become active.

Reply:
Cool, makes sense. I've changed that over in 6677ac9, as well as including a divider to better separate "Select Affected Objects" from "Disable Edit".